### ISSUE TITLE
Compiling Error in Thonny

### ISSUE THREAD

**Issue by claib** [renevb/simonsays-rpi-pico#3]
![image](https://user-images.githubusercontent.com/8424229/153999635-cdb43723-ac40-4d0b-880e-42633dd5c178.png)

**Comment by claib** [comment]
with line 11 inserted : import machine
it compiles fine
probably a thonny issue..
![image](https://user-images.githubusercontent.com/8424229/153999988-a245e0ed-2519-41d8-9bb0-6594ca858949.png)

**Comment by renevb** [comment]
Yeah, it's possible. My Thonny looks different. Probably MicroPython
versions as well. I'm glad you made it work.
